### PR TITLE
Add button to Examples to request an example

### PR DESCRIPTION
**Pull request by Taha-Hassan-Git**
This PR adds an issue template to request an example on the repo. And adds a button to the examples app to request an example.

I've added a blue background to the view code button to distinguish it from the request example. I think it makes the most sense as our primary button for the page.

### Change Type

- [ ] `patch` — Bug fix
- [ ] `minor` — New feature
- [ ] `major` — Breaking change
- [ ] `dependencies` — Changes to package dependencies[^1]
- [x] `documentation` — Changes to the documentation only[^2]
- [ ] `tests` — Changes to any test code only[^2]
- [ ] `internal` — Any other changes that don't affect the published package[^2]
- [ ] I don't know

[^1]: publishes a `patch` release, for devDependencies use `internal`
[^2]: will not publish a new version

### Release Notes

- Add a button to request an example to the examples app

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/example_request.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Need an example of how to do something with tldraw?
+  - type: textarea
+    id: description
+    attributes:
+      label: What's example?
```
Comment (mimecuvalo) [inline .github/ISSUE_TEMPLATE/example_request.yml:14]: typo: What's the example?

File: .github/ISSUE_TEMPLATE/example_request.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      description: Describe the example.
+    validations:
+      required: true
+  - type: input
+    id: contact
```
Comment (mimecuvalo) [inline .github/ISSUE_TEMPLATE/example_request.yml:19]: i don't think we need contact - it'll be through the issue, right?

**Review by mimecuvalo — COMMENTED** [review]
I think this is a good direction — I think, realistically speaking, requesting an example (or thinking of one) is a heavier lift for our team to implement (and, well, for the person to think of in the first place).

I think _above_ this button you'd want something like: [Join us on discord]
Basically, we can reduce the friction, and make it less of a dead end if you don't find what you're looking for. It also makes it less formal in most cases. I worry that Requesting an Example, if people use it, might have a high noise-to-signal — meaning that people might ask for very specific examples that are very custom to their organization or something. But I might be wrong!

(If we find people aren't using this btw, I think we probably would want to nix it after a while if it's not getting much use.)

**Comment by Taha-Hassan-Git** [comment]
> I think this is a good direction — I think, realistically speaking, requesting an example (or thinking of one) is a heavier lift for our team to implement (and, well, for the person to think of in the first place).
> 
> I think _above_ this button you'd want something like: [Join us on discord] Basically, we can reduce the friction, and make it less of a dead end if you don't find what you're looking for. It also makes it less formal in most cases. I worry that Requesting an Example, if people use it, might have a high noise-to-signal — meaning that people might ask for very specific examples that are very custom to their organization or something. But I might be wrong!
> 
> (If we find people aren't using this btw, I think we probably would want to nix it after a while if it's not getting much use.)

This makes sense, do you think not having a button to request an example makes more sense?  They could request an example in the forum after joining the discord.

**Comment by steveruizok** [comment]
I'd like to land this one today. Adding some quick social links to the sidebar. We also wanted to get rid of the regular landing page, right? I'll skip adding the links to that page.

**Pull request merged**